R/prism_to_gdrDF.R

```diff
@@ -102,6 +102,10 @@ convert_LEVEL6_prism_to_gDR_input <- function(prism_data_path,
   cell_lines <- data.table::fread(cell_line_data_path)
   treatment <- data.table::fread(treatment_data_path)
   res <- data.table::fread(prism_data_path)
+  
```

Comment:
Is column "V1" also present In public PRISM data? 👀

Reply:
Yes, when it's represented by matrix